### PR TITLE
DOCS-2954: Fix vision service code snippets

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: static/include/services/apis/generated/vision.md [hunk 1]
```diff
@@ -143,14 +147,25 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
...
+    logger.Error(err)
+    return
+}
+camStream, err := cam.Stream(context.Background())
 
 // Get an image from the camera stream
 img, release, err := camStream.Next(context.Background())
 defer release()
 
+myDetectorService, err := vision.FromRobot(machine, "my_detector")
```
Comment (sguequierre) [inline static/include/services/apis/generated/vision.md:161]: good catch

File: test-scripts/test-go/vision-service.go [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+	if err != nil {
+		logger.Fatalf("Could not get classifications: %v", err)
+	}
+	if len(classifications1) > 0 {
+		logger.Info(classifications1[0])
+	}
+
+
+
+    // mySegmenterService, err := vision.FromRobot(machine, "my_segmenter")
```
Comment (sguequierre) [inline test-scripts/test-go/vision-service.go:125]: why is this commented out?

File: test-scripts/test-vision-svc.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    my_detector = VisionClient.from_robot(machine, "my_detector")
+
+    # Get an image from the camera
+    img = await cam1.get_image()
+
+    # Get detections from that image
+    detections = await my_detector.get_detections(img)
+
+
+
```
Comment (sguequierre) [inline test-scripts/test-vision-svc.py:50]: (opt) too much space

File: test-scripts/test-vision-svc.py [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    print(f"vision-3 get_properties return value: {vision_3_return_value}")
+
+
+    import numpy as np
+    import open3d as o3d
+
+    # Grab the object segmenter you configured on your machine
+    my_segmenter = VisionClient.from_robot(machine, "my_segmenter")
+    # Get the objects from the camera output
+    # objects = await my_segmenter.get_object_point_clouds("cam")
```
Comment (sguequierre) [inline test-scripts/test-vision-svc.py:91]: again, why commenting out this method? guessing on purpose as in both code samples. did it not work?

**Review by sguequierre — APPROVED** [review]
% comments

File: test-scripts/test-vision-svc.py (hunk 4 above)
Comment (npentrel) [reply test-scripts/test-vision-svc.py:91]: I have no camera that returns point clouds, I've tested enough to know we have to pass in a camera name but can't test the rest. I'll uncomment for the PR though

File: static/include/services/apis/generated/vision.md [hunk 5]
```diff
@@ -50,8 +48,14 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
...
 ```go {class="line-numbers linkable-line-numbers"}
+myDetectorService, err := vision.FromRobot(machine, "my_detector")
+if err != nil {
+    logger.Error(err)
+    return
+}
+
 // Get detections from the camera output
-detections, err := visService.DetectionsFromCamera(context.Background(), myCam, nil)
+detections, err := visService.DetectionsFromCamera(context.Background(), "cam", nil)
```
Comment (npentrel) [inline static/include/services/apis/generated/vision.md:58]:
```suggestion
detections, err := myDetectorService.DetectionsFromCamera(context.Background(), "cam", nil)
```

**Pull request merged**